goldstone/core/models.py

```diff
@@ -2696,6 +2701,69 @@ def build_alert_template(self, hits):
         return msg_params_dict
 
 
+class AlertSearchSQLQuery(AlertSearch):
+
+    db_table = models.CharField(max_length=64,
+                                default='compliance_vulnerability',
+                                help_text='database to be searched')
+
+    db_col_filter = models.CharField(max_length=64, default='created',
```

Comment:
Explanation given in the previous comment. A re-org of these fields to be named more generically, without mentioning index or db_col is the right way to go (to be scheduled ahead ?)